### ISSUE TITLE
Abilities are not working with modded mobs

### ISSUE THREAD

**Issue by ghost** [Draylar/identity#298]
When I press R key with a modded mob that has ability, nothing happens (Mob tested: Sun, from the Dungeons Mod Lite mod)

**Comment by Draylar** [comment]
Unlike animations & visuals which can be done automatically, abilities require me (or the other developer) to add specific integration for entities. If you're looking to add abilities for a modpack and know a developer, the [AbilityRegistry](https://github.com/Draylar/identity/blob/1.17/src/main/java/draylar/identity/ability/AbilityRegistry.java) class makes it easy to integrate with Identity.

In other words, only select vanilla mobs will have abilities in Identity. A mostly complete list of supported mobs can be found [here](https://github.com/Draylar/identity/wiki/Abilities). Adding an expansion/external mod to support other projects would be possible, but is not something I plan to do.

**Comment by dhgsudgsufghdufsaiorhje9gfuhreog0kr90gd** [comment]
I tested enderman but it didn't work

**Comment by MasterOpel** [comment]
If you have enough coding knowledge, then you should be able to make a custom Identity ability for the mob(s).
